### ISSUE TITLE
Unable to import newrelic_alert_condition

### ISSUE THREAD

**Issue by davidji99** [terraform-providers/terraform-provider-newrelic#432]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
- `Terraform v0.12.20`
- `provider.newrelic >= 1.14.0`

### Affected Resource(s)
Please list the resources as a list, for example:
- newrelic_alert_condition

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
resource "newrelic_alert_condition" "response-time" {
  policy_id = data.terraform_remote_state.global.outputs.newrelic_alert_channel_ids["prod"]

  name        = "REDACTED"
  type        = "apm_app_metric"
  entities    = var.nr_service_api_entities
  metric      = "apdex"
  runbook_url = "https://www.example.com"
  condition_scope = "application"

  term {
    duration      = 10
    operator      = "below"
    priority      = "critical"
    threshold     = "5"
    time_function = "all"
  }
}
```

### Debug Output
```
newrelic_alert_condition.response-time: Importing from ID "<POLICY_ID>:<CONDITION_ID>"...
newrelic_alert_condition.response-time: Import prepared!
  Prepared newrelic_alert_condition for import
newrelic_alert_condition.response-time: Refreshing state... [id="<POLICY_ID>:<CONDITION_ID>"]

Error: json: invalid use of ,string struct tag, trying to unmarshal ".5" into float64
```

Remote API JSON:
```
{
      "id": REDACTED,
      "type": "apm_app_metric",
      "name": "REDACTED",
      "enabled": true,
      "entities": [
       some_entities
      ],
      "metric": "response_time_web",
      "runbook_url": "https://insights.newrelic.com/accounts/REDACTED/dashboards/REDACTED",
      "condition_scope": "application",
      "terms": [
        {
          "duration": "10",
          "operator": "above",
          "priority": "critical",
          "threshold": "5",
          "time_function": "all"
        }
      ]
    }
```

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
I expect the resource to be imported.

### Actual Behavior
The resource is not imported and an error is produced in my terminal.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform import newrelic_alert_condition.response-time POLICY_ID:CONDITION_ID`

### Important Factoids
We have another condition that has this:
```
      "terms": [
        {
          "duration": "15",
          "operator": "above",
          "priority": "critical",
          "threshold": ".5",
          "time_function": "all"
        }
      ],
```

I believe the API client is complaining about the `threshold` value above.

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by davidji99** [comment]
Update: I changed the `threshold` from `.5` to `0.5` in the API and the import worked properly. So either the UI shouldn't allow for non-zero, less than 1 decimals or the API client needs to be updated to support this value.

**Comment by ctrombley** [comment]
@davidji99 Thanks for the thorough writeup - this will be fixed in the next release.

**Comment by ctrombley** [comment]
Fixed in [v1.15.1](https://github.com/terraform-providers/terraform-provider-newrelic/releases/tag/v1.15.1).
